src/tools/rcll-refbox-instruct.cpp

```diff
@@ -139,18 +145,29 @@ handle_message(uint16_t                                   component_id,
 	}
 
 	std::shared_ptr<GameState> g;
-	if ((g = std::dynamic_pointer_cast<GameState>(msg)) && wait_state_) {
+	if ((g = std::dynamic_pointer_cast<GameState>(msg))) {
+		current_game_state_ = g;
+	}
+	std::shared_ptr<RobotInfo> ri;
+	if ((ri = std::dynamic_pointer_cast<RobotInfo>(msg)) && wait_state_) {
```

Comment:
the condition about `wait_state_` is not needed here

Reply:
Good point.